hack/prune-api-description-whitelist.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
```

Comment:
> Is it wrong to have? I'm going to be writing these things and it's pretty useful I will admit ...

It's one more thing to rot.  If you've gone to the trouble of writing a description, deleting a whitelist entry is minimal effort.  Especially since you have the correct messages in the verify script.